exotations/task_maps/task_map/init/JointAccelerationBackwardDifference.in

```diff
@@ -0,0 +1,8 @@
+extend <exotica/TaskMap>
+// inherited from TaskMap:
+// string Link           > name of frame in which point is defined
+// VectorXd LinkOffset   > coordinate of point in Link frame
+// string Base           > name of frame in which line is defined
+// VectorXd BaseOffset   > starting point of line in Base frame
+
+extend <exotica/PlanningProblem>
```

Comment:
Extend planning problem? Why?

Reply:
`StartState` is set in xml in `<UnconstrainedEndPoseProblem>` (or other problem type) in tags `<StartState>..</StartState>`. I am trying to expose `<StartState>` to task maps so that I can initialize the smoothing task maps using this rather than the user specify the same start state for every task map (since it can be defined below). I have tried the following:
1. Only having `extend <exotica/TaskMap>` in `.in` file. Issue: `init` does not have a member `StartState` and so compilation fails. 
1. `extend <exotica/PlanningProblem>`, this contains the optional `StartState`, gives the error: `Initializer JointVelocityBackwardDifferenceInitializer requires property PlanningScene to be set!` and so fails at run time.
1. In `.cpp` call `scene_->getModelState()`
1. In `.cpp` call `scene_->getControlledState()`
1. In `.cpp` call `scene_->getSolver().getModelState()`
1. In `.cpp` call  `scene_->getSolver().getControlledState()`

3, 4, 5, 6 all give a vector of zeros indicating to me that task maps are initialized before the problem. Only option, I can see at this point, is to force user to indicate `StartState` in the task map init by adding a `Required` parameter in the `.in` file. @VladimirIvan is it possible to expose `StartState` from problem to task maps? Or is there another way not listed above that this can already be done?